neo/Consensus/PrepareRequest.cs

```diff
@@ -23,9 +29,10 @@ public PrepareRequest()
         public override void Deserialize(BinaryReader reader)
         {
             base.Deserialize(reader);
+            Timestamp = reader.ReadUInt32();
             Nonce = reader.ReadUInt64();
             NextConsensus = reader.ReadSerializable<UInt160>();
-            TransactionHashes = reader.ReadSerializableArray<UInt256>();
+            TransactionHashes = reader.ReadSerializableArray<UInt256>(Block.MaxTransactionsPerBlock);
```

Comment:
Will it always have this size?

Reply:
No this is a maximum for deserializing.

Reply:
It will have size equal to the amount of transactions present.